docs/storage.md

```diff
@@ -1,17 +1,195 @@
+
+![Key-value store]({{@asset storage/images/key-value-overview.png}})
+
+The easiest way to access your key-value stores is via the
+[Apify app](https://my.apify.com/storage#/keyValueStores), which provides a user-friendly interface for viewing or downloading the data and editing your key-value stores' properties. 
+
+To manage the data in your key-value stores (and for more access options), you can use the
+[Apify SDK](https://sdk.apify.com/docs/api/key-value-store), Apify's [JavaScript API client](https://docs.apify.com/apify-client-js#ApifyClient-keyValueStores) or
+the [Apify API](https://docs.apify.com/api/v2#/reference/key-value-stores).
+
+For more information, see the [Key-value store]({{@link storage/key_value_store.md}}) documentation.
+
+## Request queue
+
+[Request queues]({{@link storage/request_queue.md}}) allow you to dynamically maintain a queue of URLs of web pages. You can use this in recursively crawling websites: you start from initial URLs and add new links as they are found, while skipping duplicates.
+
+![Request queue]({{@asset storage/images/request-queue-overview.png}})
+
+The easiest way to access your request queues is via the
+[Apify app](https://my.apify.com/storage#/requestQueues), which provides a user-friendly interface for viewing your request queues and editing your queues' properties. 
+
+To manage your request queues, you can use the
+[Apify SDK](https://sdk.apify.com/docs/api/request-queue), Apify's [JavaScript API client](https://docs.apify.com/apify-client-js#ApifyClient-requestQueues) or
+the [Apify API](https://docs.apify.com/api/v2#/reference/request-queues).
+
+For more information, see the [Request queue]({{@link storage/request_queue.md}}) documentation. 
+
+## Basic usage
+
+### Apify app
+
+To access your storages from the Apify app, go to the [`Storage` section](https://my.apify.com/storage) in the left-side menu. From there, you can click through the tabs to view your key-value stores, datasets, and request queues, as well as API endpoints. To view your storages, click their `ID`.
+
+![Storages in app]({{@asset storage/images/datasets-app.png}})
+
+> Only named storages are displayed by default. Select the `Include unnamed *store*` checkbox to display all of your storages.
+
+You can edit your stores' names under the `Settings` tab of their detail page. There, you can also grant [access rights](https://docs.apify.com/access-rights) to other Apify users.
+
+You can quickly share your storages' contents and details by sharing the URLs you find under the `API` tab in a store's detail page. 
+
+![Storage API]({{@asset storage/images/overview-api.png}})
+
+These URLs provide links to API `endpoints`–the places where your data are stored. Some of the endpoints do not require an [authentication token](https://docs.apify.com/api/v2#/introduction/authentication)-the calls are authenticated using a hard-to-guess ID, so they can be shared freely.
```

Comment:
I'd make this more specific - the token is not needed here for reading operations (get kv-store record, get dataset items, list kv-store keys, etc.) but is needed for any other (update, delete, ...)